### PR TITLE
Fix for the dutch postal codes

### PR DESCRIPTION
**Pull request by HDsign**
Dammit! Github sure making me look good

### CODE REVIEW

File: library/Rules/PostalCode.php [hunk 1]
```diff
@@ -113,7 +113,7 @@ class PostalCode extends Regex
         "NF" => "/^(\d{4})$/",
         "NG" => "/^(\d{6})$/",
         "NI" => "/^(\d{7})$/",
-        "NL" => "/^(\d{4}[A-Z]{2})$/",
+        'NL' => "/^(\A[1-9]\d{3} ?[a-zA-Z]{2}\z)$/",
```
Comment (henriquemoody) [inline library/Rules/PostalCode.php]: Is there any document that explains that lowercase are allowed?

Comment (HDsign) [inline library/Rules/PostalCode.php]: I haven't been able to find such information yet. The only official thing I found is this: http://www.postnl.nl/klantenservice/veelgestelde-vragen/versturen/postcodes-adressen/hoe-is-een-postcode-in-nederland-opgebouwd.html
For the rest of it, there's only Wikipedia's on the dutch zipcode. I've requested some more information on the matter by the official dutch postal company.

I can only say from experience most websites allow this regex on forms and stuff. For presentation they most of the time use this format '0000 AA'. But for form validation purposes we think it shouldn't fail.

So I'm still waiting on this official information.

Comment (henriquemoody) [inline library/Rules/PostalCode.php]: Hey @HDsign, do you have any news on it? At least the company I work in The Netherlands we don't allow lower case chars.

Comment (HDsign) [inline library/Rules/PostalCode.php]: It's PostNL, they don't provide any information on this. Never mind the change then.

**Pull request closed without merging**